app/helpers/govspeak_helper.rb

```diff
@@ -48,11 +50,16 @@ def html_attachment_govspeak_headers(attachment)
   def html_attachment_govspeak_headers_html(attachment)
     content_tag(:ol, class: ('unnumbered' if attachment.manually_numbered_headings?)) do
       html_attachment_govspeak_headers(attachment).reduce('') do |html, header|
-        html << content_tag(:li ,link_to(header.text, "##{header.id}"))
+        css_class = header_contains_manual_numbering?(header) ? 'numbered' : ''
```

Comment:
[My comment on the matching pull request](https://github.com/alphagov/whitehall/pull/1937#discussion-diff-22860678) still stands:

> rather than 'numbered' : '' could it be 'numbered' : nil. Then the govspeak_content_worker_test wouldn't need to be updated to have empty class attributes.